packages/at_onboarding_cli/lib/src/cli/auth_cli.dart

```diff
@@ -350,8 +350,16 @@ Future<void> enroll(ArgResults argResults, {AtOnboardingService? svc}) async {
   }
 
   File f = File(argResults[AuthCliArgs.argNameAtKeys]);
-  if (f.existsSync()) {
-    stderr.writeln('Error: atKeys file ${f.path} already exists');
```

Comment:
The above check should not have been removed

Reply:
For maintainability purposes. It needs to be made clear to the reader precisely what is happening here. Burying this as an exception in a method called 'isWritable' makes the code confusing